src/elements/core/testing/private/visual-regression-snapshot.ts

```diff
@@ -113,8 +113,19 @@ export const visualDiffFocus: VisualDiffState = {
   with(setup: (setup: VisualDiffSetupBuilder) => void | Promise<void>): Mocha.Func {
     return async function (this: Mocha.Context) {
       const builder = await runSetupWithViewport(setup, this.test?.ctx?.['requestViewport']);
-      builder.snapshotElement.focus();
+
+      // We create a focusable element (<a>) right before the state element.
+      // We can tab once to land on the desired element.
+      const link = document.createElement(`a`);
+      link.href = '#';
+      link.classList.add('sbb-screen-reader-only');
```

Comment:
```suggestion
```
Not sure if it's needed since we remove the element before the snapshot is taken

Reply:
You are right; it was a consideration to be sure that there is no unwanted layout shift. For 99%, we could remove it, but also after discussing it with Lukas, we decided to keep it.